### PR TITLE
Hotfix/suit 9712 Add element selector to error message

### PR DESCRIPTION
**Pull request by givehug**
Add element selector to error message

### CODE REVIEW

File: lib/chains/elementChain.js [hunk 1]
```diff
@@ -50,12 +50,30 @@ const {
...
+
+	switch (true) {
+		case !!data.selector.apiId:
+			elName = data.selector.apiId + ' ';
+			break;
+		case data.selector.video && Object.keys(data.selector).length === 1:
+			elName = 'video ';
+			break;
+		default:
+			elName = '';
```
Comment (SuitLollipop) [inline lib/chains/elementChain.js]: So what about element selected by text or by attributes is this accounted for?

File: lib/utils/stringUtils.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * @param {Array<string>} arr
+ * @param {boolean} [singleItemToNewLine] if only one item in array place it to new line or not
+ * @returns {string}
+ */
+function arrToString(arr, singleItemToNewLine) {
+	if (arr.length === 1 && !singleItemToNewLine) {
+		return `${arr[0]}`;
+	}
+
+	return arr.reduce((out, i) => out + `${EOL}\t${i}`, '');
```
Comment (JeKachan) [inline lib/utils/stringUtils.js]: @givehug why `out` not part of string template?

File: test/utils/socketErrorMessages.test.js [hunk 3]
```diff
@@ -122,6 +122,16 @@ describe('Socket error messages', () => {
...
 			[basePayload('unsupportedButton'), 'Specified buttons are not supported on this device. Chain description'],
 			[basePayload('aborted'), 'Test execution was aborted. Chain description'],
+			[
+				{
+					...basePayload('aborted'),
+					response: {
+						...basePayload('aborted').response,
+						message: {info: {reason: 'manualActionRequired'}},
+					},
+				},
```
Comment (JeKachan) [inline test/utils/socketErrorMessages.test.js]: For consistency you can replace it to something:
`
set(lensPath(['response', 'message', 'info', 'reason']), 'manualActionRequired', basePayload('aborted')),
`

File: lib/chains/elementChain.js (hunk 1 above)
Comment (givehug) [reply lib/chains/elementChain.js]: selectors list is added to error message in socketErrorMessages.js line 211

File: lib/utils/stringUtils.js (hunk 2 above)
Comment (givehug) [reply lib/utils/stringUtils.js]: updated

File: test/utils/socketErrorMessages.test.js (hunk 3 above)
Comment (givehug) [reply test/utils/socketErrorMessages.test.js]: updated

**Comment by givehug** [comment]
@andriichumak  merged

**Pull request merged**